Commands.h

```diff
@@ -185,7 +185,7 @@ unsigned long decodeIRCode() {
     delay(20);
 
     if (results.value != 0)
-      Serial.println(results.value);
```

Comment:
FYI .. what appeared to be a simple commenting-out was a major regression.

<details><summary>Try to figure out why before expanding this</summary><P/>

The `if` statement did not use curly braces `{}` to define the block that it applied to, and thus implicitly is applied to a single next statement.

When you commented out the next statement, the `if` statement then applied to the following statement, which was now `irReceiver.resume()`.

That is very unlikely to be the intent of this change.

</details>

Reply:
oh man 😅
thanks for pointing that out. this whole things is a mess. was even worse at coding back then than now

Reply:
> thanks for pointing that out. this whole things is a mess. was even worse at coding back then than now

I love that you submitted the PR!  There were many good things in there, and I'm hoping to expose the remaining gradient palettes to the UI (separate issue filed).

I called out the above only to help folks learn from real-world issues.   Noone should write code unless they expect to make mistakes.  In fact, I made a [bad PR](https://github.com/crankyoldgit/IRremoteESP8266/pull/1700) this past week, and spent 2x as long [fixing](https://github.com/crankyoldgit/IRremoteESP8266/pull/1705) it.

Allow me to share my favorite definition of "expert":
> Someone who's failed at something more often than you've succeeded.  🤣